### PR TITLE
whatis: add osx page

### PR DESCRIPTION
**Pull request by Aracki**
Simple page added. http://www.linfo.org/whatis.html

### CODE REVIEW

**Comment by sbrl** [comment]
We've already got a page for `whatis` here: https://github.com/tldr-pages/tldr/blob/master/pages/linux/whatis.md

Perhaps you could merge yours with that one?

Also, if `whatis` is available on multiple platforms, we should move it to `common` I guess.

**Comment by Aracki** [comment]
I've just investigated. I suggest I move it to `osx` dir. It seems it has totally different flags and different outputs.

**Comment by sbrl** [comment]
Cool. Does the macOS version contain any other interesting flags we could add to the page?

**Comment by Aracki** [comment]
@sbrl Linux version has multiple flags and different outputs. macOS version has no flags at all according to `man whatis`.

File: pages/osx/whatis.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# whatis
+
+> Tool that searches a set of database files containing short descriptions of system commands for keywords.
+> More information: <http://www.linfo.org/whatis.html>.
+
+- Search for informations about keyword:
```
Comment (agnivade) [inline pages/osx/whatis.md]: `informations` is grammatically incorrect. Let's also change to "about _a_ keyword"

Comment (Aracki) [reply pages/osx/whatis.md]: thanks @agnivade I didn't know `information` is non-countable word. I will make another PR for the fix.

**Pull request merged**